akka-cluster/src/main/scala/akka/cluster/Member.scala

```diff
@@ -271,17 +281,29 @@ object MemberStatus {
    */
   def removed: MemberStatus = Removed
 
+  /**
+   * Java API: retrieve the `ShuttingDown` status singleton
+   */
+  def shuttingDown: MemberStatus = PreparingForShutdown
+
+  /**
+   * Java API: retrieve the `ShutDown` status singleton
+   */
+  def shutDown: MemberStatus = ReadyForShutdown
+
   /**
    * INTERNAL API
    */
   private[cluster] val allowedTransitions: Map[MemberStatus, Set[MemberStatus]] =
     Map(
       Joining -> Set(WeaklyUp, Up, Leaving, Down, Removed),
       WeaklyUp -> Set(Up, Leaving, Down, Removed),
```

Comment:
PreparingForShutdown is not listed here but there is currently a transition WeaklyUp -> PreparingForShutdown. I suggested to remove that transition (MembershipState.allowedToPrepareToShutdown) and then this will be correct.

Reply:
Yep agreed, i forgot to remote from Weakly Up but doing that now